util/find-doc-nits

```diff
@@ -167,7 +169,10 @@ sub check()
         close POD;
     }
 
-    &check_example_location($filename, $contents) if $filename =~ m|man3/|;
+    # Check if EXAMPLES is located after RETURN VALUES section.
+    &check_section_location($filename, $contents, "RETURN VALUES", "EXAMPLES") if $filename =~ m|man3/|;
+    # Check if HISTORY is located after SEE ALSO
+    &check_section_location($filename, $contents, "SEE ALSO", "HISTORY") if $filename =~ m|man3/|;
```

Comment:
Just to be a bit extra cautious, we might want to have a check for "EXAMPLES" -> "SEE ALSO".  The checks seen here allow for the sequence "RETURN VALUES", "SEE ALSO", "EXAMPLES", "HISTORY", and I'm pretty sure we don't want that...  or?

Reply:
That's a good idea, I've pushed an extra commit with this check. Let me know if you'd like me to rebase and squash with an earlier commit.